app/containers/RegisterEmail/actions.js

```diff
@@ -83,6 +83,15 @@ export function resetState() {
   };
 }
 
+function ValidateEmail(mail)   
```

Comment:
function naming should be `someFunction()` not `SomeFunction()`, it will work fine, but let's stick with the standard naming convention.